home-manager/homemade.nix

```diff
@@ -136,4 +136,17 @@ in
       '';
     }
   )
+
+  (pkgs.writeShellApplication
+    {
+      name = "burl";
+      runtimeInputs = with pkgs; [ gh ];
+      text = ''
+        gh repo create "$1" --public --clone --template='kachick/anylang-template' --description='🚧'
```

Comment:
Adding logics here may close #410 and https://github.com/kachick/times_kachick/issues/263#issuecomment-1955792167